steps/testlio-upload/0.1.0/step.yml

```diff
@@ -0,0 +1,63 @@
+title: |-
+  Testlio Upload
+summary: |
+  Uploads IPA to Testlio
+description: |
+  Uploads IPA to Testlio to perform test case validation
+website: https://github.com/dmiluski/bitrise-step-testlio-upload
+source_code_url: https://github.com/dmiluski/bitrise-step-testlio-upload
+support_url: https://github.com/dmiluski/bitrise-step-testlio-upload/issues
+published_at: 2021-09-03T20:348:19-07:00
+source:
+  git: https://github.com/dmiluski/bitrise-step-testlio-upload.git
+  commit: 428ca8b0f6f0001181110fbcbd1b4e9f142ebd71
+host_os_tags:
+  - osx-10.10
+  - ubuntu-16.04
+project_type_tags:
+  - ios
+  - macos
+type_tags:
+  - utility
+toolkit:
+  bash:
+    entry_file: step.sh
```

Comment:
dependency on curl should be listed here

Reply:
Thanks, will update